main.go

```diff
@@ -100,7 +100,7 @@ func main() {
 	}
 
 	if configDirFlag {
-		fmt.Printf("%s\n", config.ConfigDir())
+		fmt.Printf("%s\n", config.ConfigDir("jesseduffield"))
```

Comment:
seems to me that this should use `SelectDefaultConfiguration`, does that sound right?